docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 
 services:
   hydra:
-    image: oryd/hydra:v1.0.0-rc.11
+    image: oryd/hydra:v1
```

Comment:
This is handled in #139 ?

Reply:
Yes

Reply:
It was actually not relevant to this PR but I wanted to my local development against the latest v1 image.